src/NuGet.Indexing/GalleryServiceImpl.cs

```diff
@@ -82,13 +39,13 @@ public static string QuerySearch(NuGetSearcherManager searcherManager, string q,
             }
         }
 
-        private static string DocumentCountImpl(IndexSearcher searcher, Query query, Filter filter)
+        static void DocumentCountImpl(JsonWriter jsonWriter, IndexSearcher searcher, Query query, Filter filter)
```

Comment:
It's just a nit, but we are explicit in all other code bases. Perhaps we can be consistent in cleaning a file up when we see this? :)